js/app/src/api_docs/CodeSnippet.svelte

```diff
@@ -49,35 +48,30 @@
 					<CopyButton code={python_code?.innerText} />
 				</div>
 				<div bind:this={python_code}>
-					<pre>from gradio_client import Client{#if has_file_path}, file{/if}
+					<pre><span class="highlight">from</span> gradio_client <span
+							class="highlight">import</span
+						> Client{#if has_file_path}, file{/if}
 
 client = Client(<span class="token string">"{root}"</span>)
-result = client.predict(<!--
--->{#each endpoint_parameters as { label, type, python_type, component, example_input, serializer }, i}<!--
+result = client.<span class="highlight">predict</span
+						>(<!--
+-->{#each endpoint_parameters as { python_type, example_input, parameter_name, parameter_has_default, parameter_default }, i}<!--
         -->
-		<span
-								class="example-inputs"
-								>{represent_value(example_input, python_type.type, "py")}</span
-							>,<!--
-			-->{#if dependency_failures[dependency_index][i]}<!--
-			--><span
-									class="error">ERROR</span
-								><!--
-				-->{/if}<!--
-			--><span class="desc"
-								><!--
-			-->	# {python_type.type} {#if python_type.description}({python_type.description})
-								{/if}<!---->in '{label}' <!--
-			-->{component} component<!--
-			--></span
-							><!--
-		-->{/each}<!--
+		{parameter_name
+								? parameter_name + "="
+								: ""}<span class="example-inputs"
+								>{represent_value(
```

Comment:
nit: there is an extra space added before the comma

![Pasted image 20240321162808](https://github.com/gradio-app/gradio/assets/41651716/6c319a6b-4f18-48cc-b857-514a70a69825)

Reply:
It looks like that but I think there's no space actually, maybe its a padding issue

Reply:
Interesting. I think ideal would be no padding but not a big deal